### DISCUSSION TITLE
Error connecting to API 🚨 Can't connect at all :O

### DISCUSSION THREAD

**Discussion by rphly** [supabase/supabase#2610] [Questions] [answered]
![Screenshot 2021-07-28 at 8 08 09 PM](https://user-images.githubusercontent.com/23185489/127319795-a6640bcd-6db8-4fac-abea-c3acc1682ffa.png)

Can't connect to Supabase server at all (browser, backend, other devices).

Tried restarting the database multiple times using the Restart Server function on the UI. 

Help!

**Comment by silentworks** [comment, marked as answer]
Have you try in a different browser or on a different device?

**Reply by rphly** [reply]
Hi, yeah I mentioned in the issue that I've tried different devices. seems to be one particular Supabase project and not the other.

**Reply by inian** [reply]
Reach out to beta@supabase.io with your project ref @rphly (the url in your screenshot) if you are still seeing the error.

**Reply by rphly** [reply]
Thank u @inian  :)
